test-repos/nodeJS-v2/functions.js

```diff
@@ -0,0 +1,79 @@
+module.exports = {
+  goodbye: {
+    handler: 'handler.hello',
+    events: [
+      {
+        http: {
+          path: 'goodbye',
+          method: 'get',
+          cors: true,
+          responseData: {
+            200: {
+              description: 'this went well',
+              bodyType: 'goodbye',
+            },
+          },
+          queryStringParameters: {
+            who: {
+              required: true,
+              type: 'string',
+              description: 'the person to say goodbye to',
+            },
+            count: {
+              required: false,
+              type: 'integer',
+            },
```

Comment:
We will probably want an additional parameter to cover the optional required param:
```suggestion
            },
            notRequired: {
              type: 'string',
              description: 'this param should be optional'
            },
```